packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -138,8 +138,8 @@ export default class MediaThemeMux extends MediaTheme {
           <slot name="media" slot="media"></slot>
           <media-poster-image
```

Comment:
did we want to expose this with a `poster` part?

Reply:
I'm fine with that, will add it

Reply:
upvote! 😎